app/views/invoices/_table.html.haml

```diff
@@ -12,3 +12,4 @@
   - t.col(t.sort_header(:total), class: 'right') { |i| i.decorate.total }
 
 = paginate @invoices
+= render 'multiselect_actions'
```

Comment:
This seems like it renders the multiselect-actions now below the table. IIRC, they were rendered above before. Am I missing something?